pkg/propertyprovider/azure/cloudconfig/config.go

```diff
@@ -0,0 +1,155 @@
+package cloudconfig
```

Comment:
Wantong will create a package in this repo. please use that one.

Reply:
Let's just merge this one.